src/test/java/teammates/test/cases/webapi/GetAuthInfoActionTest.java

```diff
@@ -125,9 +130,66 @@ protected void testExecute() {
         assertThrows(UnauthorizedAccessException.class, () -> getAction(new String[] {
                 Const.ParamsNames.USER_ID, "idOfAnotherInstructor",
         }));
+    }
+
+    @Test
+    public void testExecute_addCsrfTokenCookies_shouldAddToResponseAccordingToExistingCsrfToken() {
+
+        ______TS("Test will add CSRF token cookies: No logged in user");
+
+        gaeSimulation.logoutUser();
+
+        MockHttpServletRequest mockRequest = new MockHttpServletRequest(getRequestMethod(),
+                Const.ResourceURIs.URI_PREFIX + getActionUri());
+        mockRequest.addCookie(new Cookie(Const.CsrfConfig.TOKEN_COOKIE_NAME, "some random csrf token"));
+        MockHttpServletResponse mockResponse = new MockHttpServletResponse();
+
+        GetAuthInfoAction a = getAction(mockRequest, getRequestMethod(), mockResponse);
```

Comment:
You can add a new method `getHttpServletRequest()` in `GetAuthInfoAction` to get the mocked request.